specification/search/data-plane/Azure.Search/preview/2023-10-01-Preview/searchindex.json

```diff
@@ -2596,6 +2604,7 @@
           "description": "The comma-separated list of field names used for semantic ranking."
         },
         "vectorQueries": {
+          "x-ms-client-name": "vectorizableQueries",
```

Comment:
Could we just call this `queries`?

Reply:
I don't believe we expose this type publicly in the SDKs, @xiangyan99, @dgetu, and @ShivangiReja please correct me if I'm wrong. But if we do, I don't think `SearchRequest.queries` is very clear on what it's for and dropping back down to `vectorQueries` ties too closely to the `RawVectorQuery` -> `VectorQuery` rename which could lead to thinking only `float[]`-based vector querying is allowed.

Reply:
If it doesn't show up in our SDK than why are we putting a client name on it?

Reply:
It doesn't show up in Java but I'm not 100% certain on JS, Python, and .NET.

Also, I forgot to add that SearchRequest also has a new option `semanticQuery` in `2023-10-01-Preview` which using `queries` makes more confusing as well.

Reply:
The TS SDK doesn't expose this directly, but we do expose an authored type `SearchRequestOptions`, which contains conveniences and the vector and semantic parameter groupings. When we coerce the authored type to the generated type or vice-versa, we only have to explicitly declare properties when their names differ. Typically, having `x-ms-client-name` match the name in the authored type makes things a bit easier on our end. However, since we're dropping `vector`/`vectorizable` in the vector parameter grouping (correct me if I'm wrong), it makes no difference on our end whether we specify `x-ms-client-name` in this case.

Reply:
We don't expose `SearchRequest.queries` publically, now that we have defined `SemanticSearchOptions` and `VectorSearchOptions` within `SearchOptions` in SDK. In custom layer, we can call this `SearchOptions.VectorSearch.Queries` in SDK.

I believe we don't need to add  "x-ms-client-name" here.

Reply:
This has been removed.